### PR TITLE
Add instrumentation to help track down connection issues

### PR DESCRIPTION
**Pull request by Atlante45**
https://highfidelity.atlassian.net/browse/BUGZ-110
https://highfidelity.atlassian.net/browse/BUGZ-128

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19293/HighFidelity-Beta-PR15529-5c42838.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19293/HighFidelity-Beta-Interface-PR15529-5c42838.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19293/HighFidelity-Beta-PR15529-5c42838.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19293/HighFidelity-Beta-Interface-PR15529-5c42838.exe

File: domain-server/src/DomainServer.cpp [hunk 1]
```diff
@@ -1606,9 +1603,9 @@ void DomainServer::handleFailedICEServerAddressUpdate(QNetworkReply* requestRepl
     } else {
         const int ICE_SERVER_UPDATE_RETRY_MS = 2 * 1000;
 
-        qWarning() << "Failed to update ice-server address with High Fidelity Metaverse - error was"
+        qCWarning(domain_server_ice) << "Failed to update ice-server address with High Fidelity Metaverse - error was"
                    << requestReply->errorString();
```
Comment (SimonWalton-HiFi) [inline domain-server/src/DomainServer.cpp]: Maybe these and similar messages could give the ICE address (_iceServerAddr).

File: libraries/networking/src/DomainHandler.cpp [hunk 2]
```diff
@@ -457,10 +473,10 @@ void DomainHandler::processICEResponsePacket(QSharedPointer<ReceivedMessage> mes
...
     if (_icePeer.getUUID() != _pendingDomainID) {
-        qCDebug(networking) << "Received a network peer with ID that does not match current domain. Will not attempt connection.";
+        qCDebug(networking_ice) << "Received a network peer with ID that does not match current domain. Will not attempt connection.";
         _icePeer.reset();
     } else {
-        qCDebug(networking) << "Received network peer object for domain -" << _icePeer;
+        qCDebug(networking_ice) << "Received network peer object for domain -" << _icePeer;
 
         // ask the peer object to start its ping timer
         _icePeer.startPingTimer();
```
Comment (SimonWalton-HiFi) [inline libraries/networking/src/DomainHandler.cpp]: Can we add more logging to `DomainHandler::checkInPacketTimeout()`, such as when `_checkInPacketsSinceLastReply` > 1, to track the disconnects.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19308/HighFidelity-Beta-PR15529-c691ee4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19308/HighFidelity-Beta-Interface-PR15529-c691ee4.dmg

**Comment by SimonWalton-HiFi** [comment]
Looks good!

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19308/HighFidelity-Beta-PR15529-c691ee4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19308/HighFidelity-Beta-Interface-PR15529-c691ee4.exe

**Comment by SimonWalton-HiFi** [comment]
Do we have diagnostics for when `LimitedNodeList::removeSilentNodes()` kills a Node? It seems like the ACs at least use this routine.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19319/HighFidelity-Beta-PR15529-07372f1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19319/HighFidelity-Beta-Interface-PR15529-07372f1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19319/HighFidelity-Beta-PR15529-07372f1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19319/HighFidelity-Beta-Interface-PR15529-07372f1.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19319/HighFidelity-Beta-PR15529-07372f1.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19319/HighFidelity-Quest-Beta-PR15529-07372f1.apk)

**Pull request merged**